doc/articles/getting-started/requirements.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Uno Platform requirements
+---
+
+The platform requirements for the Uno Platform are as follows:
+
+
+## Android
+
+- The minimumn supported version by the Uno Platform is Android "Lollipop" 5.0 (API 21) from 2014.
```

Comment:
- Android v4.1 currently does not work, see 
https://github.com/nventive/Uno/issues/20
- Android v4.4 has a lot of limitations that make it hard to work with. 

We could either have the minver as v5.0 or change the language here to be:

- The minimum supported version by the Uno Platform is Android "Lollipop" 5.0 (API 21) from 2014. Android v4.4 is known to work but is unsupported.

Yay/Nay?

Reply:
Second part is correct.